driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideOperationsTimeoutProseTest.java

```diff
@@ -742,10 +732,53 @@ public void shouldIgnoreWtimeoutMsOfWriteConcernToInitialAndSubsequentCommitTran
             if (command.containsKey("writeConcern")) {
                 BsonDocument writeConcern = command.getDocument("writeConcern");
                 assertFalse(writeConcern.isEmpty());
-                assertFalse(writeConcern.containsKey("wTimeoutMS"));
+                assertFalse(writeConcern.containsKey("wtimeout"));
             }});
     }
 
+    /**
+     * Not a prose spec test. However, it is additional test case for better coverage.
+     */
+    @Test
+    @DisplayName("Should refresh timeout for commit transaction")
```

Comment:
How does this test check that the timeout is refreshed?  Seems to just confirm that the second commit fail with MongoOperationTimeoutException?

Reply:
I have corrected the display name to be `Should throw timeout exception for subsequent commit transaction`. Good catch!